daemon/mgr/spec_mount.go

```diff
@@ -83,3 +96,78 @@ func setupMounts(ctx context.Context, c *Container, s *specs.Spec) error {
 	}
 	return nil
 }
+
+// generateNetworkMounts will generate network mounts.
+func generateNetworkMounts(c *Container) []specs.Mount {
+	mounts := make([]specs.Mount, 0)
+
+	// if disable hostfiles, we will not mount the host files into container.
+	if c.Config.DisableHostsfiles {
+		return mounts
+	}
+
+	if c.HostnamePath != "" {
```

Comment:
there are many duplicate codes here, maybe we can use 
```
for _, bind := range []pathBinding{{Src: xxx, Dst: xxx}} {
}
```

WDYT? :)

Reply:
agree it